Jenkinsfile

```diff
@@ -134,6 +134,19 @@ pipeline {
         //     }
         //   }
         // }
+        stage('Quality Check (PR)') {
+            options {
+                timeout(time: 30, unit: 'MINUTES') // timeout on this stage
+            }
+            when { expression { ( BRANCH_NAME != 'develop' ) } }
+            agent { label 'code-tests' }
+            steps {
+                script {
+                    checkout scm
+                    sh "./player.sh scan"
+                }
+            }
+        }
```

Comment:
This is a temporary addition to verify that the changes made in this PR work correctly on SonarQube. When approved, it will be removed prior to its merge to develop.

Reply:
This code block has now been removed.